includes/admin/class-admin.php

```diff
@@ -1162,6 +1175,39 @@ private function check_deprecation_warnings() {
             }
         }
 
+		/**
+		 * Check old inline fontawesome setting.
+		 * If it was enabled by default, we show an admin notice.
+		 * On dismiss we keep the setting.
+		 *
+		 * @since x.x
+		 */
+		private function check_font_awesome_setting() {
+			if ( ! current_user_can( 'install_plugins' ) || ! WPBDP_App_Helper::is_bd_page() ) {
+				return;
+			}
+
+			if ( ! wpbdp_get_option( 'enqueue-fontawesome-styles', false ) ) {
+				return;
+			}
+
+			if ( is_plugin_active( 'font-awesome/index.php' ) && ( class_exists( '\FortAwesome\FontAwesome_Loader' ) ) ) {
+				wpbdp_set_option( 'enqueue-fontawesome-styles', false );
+				return;
+			}
+
+			$this->messages[] = array(
+				sprintf(
+					_x( 'We changed how %1$s is integrated into Business Directory. Go %2$shere%3$s to install the official "%1$s styles" plugin.', 'admin', 'business-directory-plugin' ),
+					'FontAwesome',
```

Comment:
I don't see a reason to keep this out of the translatable string since we are translating it in other spots. Also, please make sure we are using "Font Awesome" consistently with a space.